app/scripts/check_requirements

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+/**
+ * Colorize the shell output
+ *
+ * @param string $text The text.
+ * @param string $color Color of the text: green, red or nothing.
+ * @return string
```

Comment:
nit: I would add an empty line before this one, or remove the one in the next function